src/containers/Tokens/index.js

```diff
@@ -49,7 +49,7 @@ const Tokens = props => {
   ) : (
     <div className="token-discovery-page">
       <TokensHeader tokens={allTokens} isLoading={isLoading} isError={isError} />
-      <TokensTable allTokens={allTokens} isError={isError} />
+      <TokensTable allTokens={allTokens} isError={isError} t={t} />
```

Comment:
I had to add this otherwise `t` is undefined.
Did you encounter this behaviour before?